examples/minimal-mdns/client.cpp

```diff
@@ -213,7 +213,7 @@ void BroadcastPacket(mdns::Minimal::ServerBase * server)
     QuerySplitter query;
     query.Split(gOptions.query);
 
-    mdns::Minimal::QueryBuilder builder(buffer.Get_ForNow());
+    mdns::Minimal::QueryBuilder builder(buffer.Retain());
```

Comment:
Hmm.  It seems like we could make the builder take a const ref to PacketBufferHandle (and have its member be a const ref) and avoid the extra refcounting  And here we could scope it so it goes out of scope before we do the `BroadcastSend` call...

Reply:
Ah, yes. I probably had TLV's chaining in mind when I converted this one. Fixed.